x/upgrade/types/proof.go

```diff
@@ -0,0 +1,12 @@
+package types
+
+import codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+
+// ProofInfo contains the upgraded client and consensus state along with the associated proofs
+// This will be useful for relayers who want to upgrade IBC clients of this chain on other counterparty chains
+type ProofInfo struct {
+	ClientState    *codectypes.Any `json:"client_state"`
```

Comment:
These use the types that are required by upgrade client msg so relayers have to do no converting